src/Mod/TechDraw/Gui/QGSPage.cpp

```diff
@@ -147,10 +138,47 @@ void QGSPage::mouseReleaseEvent(QGraphicsSceneMouseEvent * event)
     }
 
     QGraphicsScene::mouseReleaseEvent(event);
+}
+
+//! returns true if clicking on the item should clear the selection
+bool QGSPage::itemClearsSelection(int itemTypeIn)
+{
+    // type 13 is the itemUnderMouse on a page outside of any views. It is not
+    // the template or background or foreground.  QGraphicsItem type = 13 is not
+    // documented and not found in QGraphicsItem.h.
+    const std::vector<int> ClearingTypes { 13,                        // MysteryType
+                                     QGraphicsItem::UserType + 150,   // QGITemplateType
+                                     QGraphicsItem::UserType + 151,   // QGIDrawingTemplateType
+                                     QGraphicsItem::UserType + 153 }; // QGISVGTemplateType
```

Comment:
To do; move type definitions to global file

Reply:
Yeah, that's been on my list forever.